tests/unit/plugins/test_go.py

```diff
@@ -110,6 +66,30 @@ def test_schema(self):
                 "Expected {!r} to be included in properties".format(expected),
             )
 
+        # Check go-channel
+        go_channel = properties["go-channel"]
+        for expected in ["type", "default"]:
+            self.assertTrue(
+                expected in go_channel,
+                "Expected {!r} to be included in 'go-channel'".format(expected),
+            )
+
+        go_channel_type = go_channel["type"]
+        self.assertThat(
+            go_channel_type,
+            Equals("string"),
+            'Expected "go-channel" "type" to be "string", but '
+            'it was "{}"'.format(go_channel_type),
+        )
+
+        go_channel_default = go_channel["default"]
+        self.assertThat(
+            go_channel_default,
+            Equals("latest/stable"),
+            'Expected "go-channel" "default" to be '
+            '"latest/stable", but it was "{}"'.format(go_channel_default),
```

Comment:
... and we're also checking for double quotes here. (fin)

These three comments of mine are more a question of should these messages be unified in their usage of quotes? Here we're checking the output in a test so they need to match what is actually being output, but that output should IMO be consistent with it's usage of souble vs single quotes. Maybe make a note to go back and re-do those in a separate PR?

Reply:
Yes, they should and yes they are, this is just test code though with brief summaries of why things fail which should not reach an end user :smile: